packages/features/ee/organizations/pages/settings/profile.tsx

```diff
@@ -183,6 +183,11 @@ const OrgProfileForm = ({ defaultValues }: { defaultValues: FormValues }) => {
 
   const mutation = trpc.viewer.organizations.update.useMutation({
     onError: (err) => {
+      // Handle JSON parsing errors from body size limit exceeded
+      if (err.message.includes("Unexpected token") && err.message.includes("Body excee")) {
```

Comment:
The unhandled error is: "Unexpected token 'B', "Body excee"... is not valid JSON"
So this takes care of that.